packages/web-forms/src/demo/FormUpload.vue

```diff
@@ -150,20 +149,22 @@ document.addEventListener(
 				<template	v-if="!formUploading">
 					<input ref="fileInput" type="file" hidden accept=".xls, .xlsx, .xml" @change="fileChangeHandler">
 					<span>
-						<span class="icon-insert_drive_file" />
+						<span class="icon-doc-file">
```

Comment:
Changing the icon for better recognition and fit with the new style 
I know this isn't the nicest to have inline, something [I want to fix a long with this PR](https://github.com/getodk/web-forms/pull/279)